src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -28,7 +31,7 @@ class ResourceService(private val resourceTypes: Map[ResourceTypeName, ResourceT
     Future.successful(resourceTypes.get(name))
   }
 
-  def createResourceType(resourceType: ResourceType): Future[ResourceTypeName] = {
+  def createResourceType(resourceType: ResourceType): IO[ResourceTypeName] = {
```

Comment:
thanks for review!...yea..it's not consistent right now...I touched those that I need to interact with, but left the rest as is. I'll probably convert more in separate PRs just in case I accidentally change semantics